buffer/buffer.go

```diff
@@ -44,6 +44,11 @@ func (b *Buffer) AppendString(s string) {
 	b.bs = append(b.bs, s...)
 }
 
+// AppendBytes appends a byte slice to the Buffer.
+func (b *Buffer) AppendBytes(bs []byte) {
+	b.bs = append(b.bs, bs...)
+}
```

Comment:
Why create this method instead of just using Write?

Since this package and type is public, we'd prefer to not introduce new
methods if possible.

Reply:
Just figured it helped avoid checking for errors via `Write()` which were known-impossible, but not worth disrupting the larger picture just for that (and in a previous, unsubmitted version, I was calling it from more than one site, so it was a mild ergonomic improvement).